### DISCUSSION TITLE
Bring back Light Mode

### DISCUSSION THREAD

**Discussion by danieltorscho** [quasarframework/quasar#16551] [Ideas / Proposals]
My eyes are hurt when I use dark mode. Please bring back "light mode" button at least.

**Comment by pdanpdan** [comment]
`Ars Enixa Est Candelam`
Where?

**Reply by danieltorscho** [reply]
The whole website has disappearing light/dark mode switch. It randomly appears there from time to time.

pics:
![image](https://github.com/quasarframework/quasar/assets/726563/e11a4bbf-333f-4fac-9ccb-9b1200f7dcc7)
![image](https://github.com/quasarframework/quasar/assets/726563/34504e6c-3640-46e4-ab66-1c7a5de51abb)
![image](https://github.com/quasarframework/quasar/assets/726563/4e6d7468-271f-4a58-ae57-4088890aebfe)

**Comment by pdanpdan** [comment]
There are a few pages that are only in dark mode. I highly doubt you'll read the landing page or the components index so much add to make your eyes hurt :)
